pkg/image/controller/controller.go

```diff
@@ -39,35 +33,105 @@ const retryCount = 2
 // the image stream is not modified (so it will be tried again later). If a permanent
 // failure occurs the image is marked with an annotation. The tags of the original spec image
 // are left as is (those are updated through status).
+// There are 3 use cases here:
+// 1. spec.DockerImageRepository defined without any tags results in all tags being imported
+//    from default registry
+// 2. spec.DockerImageRepository + tags defined - import only the provided tags,
+//    for kind == DockerImage import underlying image, iow. exact tag (no provided means latest),
+//    for kind != DockerImage import tag from the default registry
```

Comment:
We've talked yesterday about the implicit policy, the idea was use the default and just specify a list of tags you want to import without the need to specify the underlying image each time. But your solution sounds ok to me, unless @ncdc agrees as well, since he was voicing the need to have implicit import policy.